packages/toolpad-app/src/toolpad/AppEditor/GlobalScopeExplorer.tsx

```diff
@@ -54,10 +54,28 @@ function groupScopeMeta(value: Record<string, unknown>, meta: ScopeMeta): Explor
     structure[group].items.push({
```

Comment:
Could we just `.filter` out the env bindings from `structure[group].items` instead so that they don't show at all?